docs/site/BelongsTo-relation.md

```diff
@@ -0,0 +1,180 @@
+  `BelongsToAccessor<targetModel, typeof sourceModel.prototype.id>` on the
+  source repository class.
+- call the `_createBelongsToAccessorFor` function in the constructor of the
+  source repository class with the relation name (decorated relation property on
+  the source model) and target repository instance and assign it the property
+  mentioned above.
+
+The following code snippet shows how it would look like:
+
+{% include code-caption.html
+content="/src/repositories/order.repository.ts.ts" %}
+
+```ts
+import {Getter, inject} from '@loopback/context';
+import {
+  BelongsToAccessor,
+  DefaultCrudRepository,
+  juggler,
+  repository,
+} from '@loopback/repository';
+import {Customer, Order} from '../models';
+import {CustomerRepository} from '../repositories';
+
+export class OrderRepository extends DefaultCrudRepository<
+  Order,
+  typeof Order.prototype.id
+> {
+  public readonly customer: BelongsToAccessor<
+    Customer,
+    typeof Order.prototype.id
+  >;
+
+  constructor(
+    @inject('datasources.db') protected db: juggler.DataSource,
+    @repository.getter('CustomerRepository')
+    customerRepositoryGetter: Getter<CustomerRepository>,
+  ) {
+    super(Order, db);
+    this.customer = this._createBelongsToAccessorFor(
+      'customerId',
+      customerRepositoryGetter,
+    );
+  }
+}
+```
+
+`BelongsToAccessor` is a function accepting the primary key (id) of a source
+model instance (e.g. `order.id`) and returning back the related target model
+instance (e.g. a `Customer` the order belongs to). See also
+[API Docs](https://apidocs.strongloop.com/@loopback%2fdocs/repository.html#BelongsToAccessor)
+
+## Using BelongsToAccessor in a controller
+
+The same pattern used for ordinary repositories to expose their CRUD APIs via
+controller methods is employed for `belongsTo` relation too. Once the belongsTo
+relation has been defined and configured, a new controller method can expose the
+accessor API as a new endpoint.
+
+{% include code-caption.html content="src/controllers/order.controller.ts" %}
+
+```ts
+import {repository} from '@loopback/repository';
+import {get} from '@loopback/rest';
+import {Customer, Order} from '../models/';
+import {OrderRepository} from '../repositories/';
+
+export class OrderController {
+  constructor(
+    @repository(OrderRepository) protected orderRepository: OrderRepository,
+  ) {}
+
+  // (skipping regular CRUD methods for Order)
+
+  @get('/orders/{id}/customer')
+  async getCustomer(
+    @param.path.number('id') orderId: typeof Order.prototype.id,
+  ): Promise<Customer> {
+    return await this.orderRepository.customer(orderId);
+  }
+}
+```
+
+In LoopBack 3, the REST APIs for relations were exposed using static methods
+with the name following the pattern `__{methodName}__{relationName}__` (e.g.
+`Order.__get__customer`). While we recommend to create a new controller for each
+hasMany relation in LoopBack 4, we also think it's best to use the main CRUD
```

Comment:
hasMany --> belongsTo?

Reply:
LOL, I should run a search for "hasMany" after creating this page as a copy of HasMany-relation.md.

Thank you for catching this.

Reply:
Actually, "hasMany" is correctly used here. The full sentence:

> While we recommend to create a new controller for each hasMany relation in LoopBack 4, we also think it's best to use the main CRUD controller as the place where to explose `belongsTo` API.

Reply:
Ah I see :-p